brooklyn-docs/guide/start/managing.md

```diff
@@ -3,68 +3,357 @@ title: Monitoring and Managing Applications
 title_in_menu: Monitoring and Managing Applications
 layout: website-normal
 menu_parent: index.md
+children:
+- { section: Applications } 
+- { section: Entities } 
+- { section: Sensors  } 
+- { section: Effectors  } 
+- { section: Activities } 
 ---
 
-Click on the application name, or open the Applications tab.
 
-We can explore the management hierarchy of the application, which will show us the entities it is composed of.
 
- * My Web Cluster (A `BasicApplication`)
-     * My DB (A `MySqlNode`)
-     * My Web (A `ControlledDynamicWebAppCluster`)
-        * Cluster of TomcatServer (A `DynamicWebAppCluster`)
-        * NginxController (An `NginxController`)
+So far we have touched on Brooklyn's ability to *deploy* an application blueprint to a cloud provider, but this just 
+the beginning. The sections below outline how to manage the application that has been deployed.
 
 
+## Scopes in CLI commands
```

Comment:
I think that most of the occurences of "text" on this page would be better as \`text\`.